packages/client/src/components/VisualStyles.test.tsx

```diff
@@ -1,56 +1,66 @@
 import "jest-styled-components";
 
+import { captureException } from "@sentry/browser";
```

Comment:
Hmmm. I didn't know there were two test files for `Visual`. This tests in this file can be merged into in `Visual.test.tsx`. Some tetst are very similar and can be replaced.